plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4468,6 +4485,25 @@ else if (mechanism.requireObject(EntityTag.class)) {
                 chiseledBookshelf.setLastInteractedSlot(input.asInt() - 1);
             }
         });
+
+        // <--[mechanism]
+        // @object LocationTag
+        // @name page
+        // @input ElementTag(Number)
+        // @description
+        // Changes the page currently displayed on the book in a lectern block.
+        // @tags
+        // <LocationTag.page>
+        // -->
+        tagProcessor.registerMechanism("page", false, ElementTag.class, (object, mechanism, input) -> {
+            if (object.getBlockState() instanceof Lectern lectern) {
+                lectern.setPage(input.asInt() - 1);
+                object.getBlockState().update();
```

Comment:
Need to `Mechanism#requireInteger` in here, probably as an early return before the state checks